components-compose/src/main/java/uk/gov/hmrc/components/compose/atom/button/Button.kt

```diff
@@ -98,29 +101,31 @@ fun PrimaryButton(
     )
 }
 
-@Composable
-fun SecondaryButton(
-    text: String,
-    modifier: Modifier = Modifier,
-    enabled: Boolean = true,
-    textAlign: TextAlign = TextAlign.Center,
-    buttonColors: ButtonColors = ButtonDefaults.buttonColors(
-        containerColor = Color.Transparent, contentColor = colors.hmrcBlue
-    ),
-    onClick: () -> Unit,
-    content: @Composable RowScope.() -> Unit = {},
-) {
-    CompositionLocalProvider(LocalRippleTheme provides HmrcRippleTheme) {
-        HmrcButton(
-            text = text,
-            modifier = modifier,
-            contentPadding = PaddingValues(all = HmrcTheme.dimensions.hmrcSpacing16),
-            enabled = enabled,
-            textAlign = textAlign,
-            buttonColors = buttonColors,
-            onClick = onClick,
-        ) {
-            content()
+object SecondaryButton {
+    @Composable
+    operator fun invoke(
+        text: String,
+        modifier: Modifier = Modifier,
+        enabled: Boolean = true,
```

Comment:
I don't think we need to define this here, so we may as well tidy up now we're here. The `enabled` field now has a default value of true in `HmrcButton` and there isn't a use case where we would want to have a disabled secondary button, so we can tidy up and remove this, and just rely on the default that's built in.
```suggestion
```